doc/guide/dynamics/dynamics-monte.rst

```diff
@@ -203,110 +339,111 @@ For instance, consider the following T1 simulation of a qubit with a lifetime of
-    plt.plot(times, data1.expect[0], times, data1.expect[1], lw=2)
-    plt.plot(times, data2.expect[0], '--', times, data2.expect[1], '--', lw=2)
-    plt.plot(times, data_merged.expect[0], ':', times, data_merged.expect[1], ':', lw=2)
-    plt.title('Monte Carlo time evolution')
-    plt.xlabel('Time', fontsize=14)
-    plt.ylabel('Expectation values', fontsize=14)
-    plt.legend(("cavity photon number", "atom excitation probability"))
-    plt.show()
+.. code-block::
 
+    >>> print(res_par.seeds[:3])
+    [SeedSequence(entropy=1,spawn_key=(1,),),
+     SeedSequence(entropy=1,spawn_key=(0,),),
+     SeedSequence(entropy=1,spawn_key=(2,),)]
 
-This can be used to explore the convergence of the Monte Carlo solver.
-For example, the following code block plots expectation values for 1, 10 and 100 trajectories:
+    >>> print(res_ser.seeds[:3])
+    [SeedSequence(entropy=1,spawn_key=(0,),),
+     SeedSequence(entropy=1,spawn_key=(1,),),
+     SeedSequence(entropy=1,spawn_key=(2,),)]
 
-.. plot::
-    :context: close-figs
 
-    solver = MCSolver(H, c_ops=[np.sqrt(0.1) * a])
+Photocurrent
+------------
 
-    data1 = solver.run(psi0, times, e_ops=[a.dag() * a, sm.dag() * sm], ntraj=1)
-    data10 = data1 + solver.run(psi0, times, e_ops=[a.dag() * a, sm.dag() * sm], ntraj=9)
-    data100 = data10 + solver.run(psi0, times, e_ops=[a.dag() * a, sm.dag() * sm], ntraj=90)
+The photocurrent, previously computed using the ``photocurrent_sesolve`` and
+``photocurrent_sesolve`` functions, are now included in the output of
+:func:`.mcsolve` as ``result.photocurrent``.
 
-    expt1 = data1.expect
-    expt10 = data10.expect
-    expt100 = data100.expect
+
+.. plot::
+    :context: close-figs
+
+    times = np.linspace(0.0, 10.0, 200)
+    psi0 = tensor(fock(2, 0), fock(10, 8))
+    a  = tensor(qeye(2), destroy(10))
+    sm = tensor(destroy(2), qeye(10))
+    e_ops = [a.dag() * a, sm.dag() * sm]
+    H = 2*np.pi*a.dag()*a + 2*np.pi*sm.dag()*sm + 2*np.pi*0.25*(sm*a.dag() + sm.dag()*a)
+    data = mcsolve(H, psi0, times, [np.sqrt(0.1) * a], e_ops=e_ops)
 
     plt.figure()
-    plt.plot(times, expt1[0], label="ntraj=1")
-    plt.plot(times, expt10[0], label="ntraj=10")
-    plt.plot(times, expt100[0], label="ntraj=100")
-    plt.title('Monte Carlo time evolution')
+    plt.plot((times[:-1] + times[1:])/2, data.photocurrent[0])
+    plt.title('Monte Carlo Photocurrent')
     plt.xlabel('Time')
-    plt.ylabel('Expectation values')
-    plt.legend()
+    plt.ylabel('Photon detections')
     plt.show()
 
+
 .. openmcsolve:
 
 Open Systems
 ------------
 
-``mcsolve`` can be used to study system with have measured and dissipative interaction with the bath.
-This is done by using a liouvillian including the dissipative interaction instead of an Hamiltonian.
+``mcsolve`` can be used to study system with have measured and dissipative
+interaction with the bath.  This is done by using a liouvillian including the
+dissipative interaction instead of an Hamiltonian.
```

Comment:
```suggestion
``mcsolve`` can be used to study systems which have measurement and dissipative
interactions with their environment.  This is done by passing a Liouvillian including the
dissipative interaction to the solver instead of a Hamiltonian.
```